src/routes/survey.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import Navbar from 'react-bootstrap/Navbar';
+import back from '../assets/back.png';
+import SurveyForm from '../components/survey-form';
+import { Link } from "react-router-dom";
+
+const Topbar = styled(Navbar)`
```

Comment:
It looks like this is repeated several times. We might want to consider making this a separate component. On the other hand, is it necessary to used styled components here if all you're doing is changing the background color?

Reply:
re: repeat, yea would be a good idea to refactor and have props that indicate whether there should be a back button + what the name of the top bar is + link to where the back goes. might refactor this in the future some time

re: changing the background color, i think it's just easier to use styled components so it's easier to have the css defined up top rather than inline although yeah im a little inconsistent about that